### ISSUE TITLE
No name is defined for "How can I write unit tests for python function" button: A11y_VS Code extension for API Center_Copilot Chat_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#100]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG4.1.2; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Name Role Value; #AILimited; #A11yMAS;#FirstParty:Bug212323;#External;#External:VSCode;#BM-VisualStudioCodeClient-Win32-Jan2024;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:

1. Create a API center in azure portal.
2. Open Visual studio Code.
3. Tab till Chat option in the left side panel and press enter.
4. Type @workspace /tests in the copilot chat box.
5. Verify whether the name is defined for "How can I write unit tests for python function" button or not.

### Actual Result:
No name is defined for "How can I write unit tests for python function" button.

### Expected Result:
Name should be defined for "How can I write unit tests for python function" button.

### User Impact:
Screen reader users will not get the information about the functionality of the button if name is not defined.

### Attachment: 
![Bug_100](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/b4167511-c7f3-4555-98d2-c2674c90ce35)

**Comment by formulahendry** [comment]
This issue is caused by UI of GitHub Copilot Chat extension, not related to UI of API Center extension.

**Comment by formulahendry** [comment]
Opened bug in vs code repo: https://github.com/microsoft/vscode/issues/212323

**Comment by AnKushSingh05** [comment]
#External; #FirstParty:Bug212323; #External:VS Code;

**Comment by shindepra** [comment]
Verified the issue in below environment
Version: 1.89.1 (user setup)
Issue is fixed. Hence closing the bug.
![image](https://github.com/microsoft/vscode-azureapicenter/assets/143686145/16edfe4f-da52-4579-af8a-312133c443d7)
